qemu/tests/cfg/uefi_check_secure_MOR.cfg

```diff
@@ -0,0 +1,22 @@
+- uefi_check_secure_MOR:
+    only q35
+    only ovmf
+    only Win2016, Win10
+    type = uefi_check_secure_MOR
+    timeout = 360
+    start_vm = no
+    restore_ovmf_vars = yes
+    auto_cpu_model = no
+    cpu_model = host
+    check_secure_boot_enabled_cmd = 'powershell -command "Confirm-SecureBootUEFI"'
+    dgreadiness_tool = '/home/dgreadiness.zip'
+    src_path = '/home/dgreadiness_v3.6'
+    dst_path = "C:\dgreadiness"
+    dgreadiness_tool_path = "cd ${dst_path}\dgreadiness_v3.6"
```

Comment:
can we change the name? for this is really a cmd instead of a path

Reply:
Thank you very much, updated the code.
Could you help review it again?